cmd/bootstrapper/initialize_kubelet.go

```diff
@@ -51,6 +53,9 @@ func init() {
 	initializeKubeletCmd.PersistentFlags().StringVar(&initializeKubeletOpts.nodeIP, "node-ip", "",
 		"nodeIP is the IP that should be used as the node object's IP. "+
 			"If unset, kubelet will determine the IP itself.")
+	initializeKubeletCmd.PersistentFlags().StringVar(&initializeKubeletOpts.clusterDNSIP, "cluster-dns-ip", "",
+		"The DNS server IP which will be used by all containers. "+
+			"If unset, kubelet will sue the host's DNS servers")
```

Comment:
You need say how WMCB is going to use this field and not how the kubelet uses it. This also needs to be reflected in the README.